tests/apps/perf/actorfeatures/app.go

```diff
@@ -46,23 +47,28 @@ type daprConfig struct {
 	DrainRebalancedActors   bool     `json:"drainRebalancedActors,omitempty"`
 }
 
-var registeredActorType = getActorType()
+var registeredActorType = map[string]bool{}
 
 var daprConfigResponse = daprConfig{
-	[]string{getActorType()},
+	getActorType(),
 	actorIdleTimeout,
 	actorScanInterval,
 	drainOngoingCallTimeout,
 	drainRebalancedActors,
 }
 
-func getActorType() string {
+func getActorType() []string {
 	actorType := os.Getenv(actorTypeEnvName)
 	if actorType == "" {
-		return defaultActorType
+		registeredActorType[defaultActorType] = true
+		return []string{defaultActorType}
 	}
 
-	return actorType
+	actorTypes := strings.Split(actorType, ",")
+	for _, tp := range actorTypes {
+		registeredActorType[tp] = true
+	}
+	return actorTypes
```

Comment:
I am assuming there is no concurrency concern here because the worst that could happen is a race condition to set the same key to true, correct?